recipes/podofo/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "5":
+            raise ConanInvalidConfiguration("PoDoFo requires GCC >=5")
+
+    def requirements(self):
+        self.requires("freetype/2.10.4")
+        if self.settings.os == "Linux":
+            self.requires("fontconfig/2.13.92")
+        self.requires("libjpeg/9d")
+        self.requires("libunistring/0.9.10")
+        self.requires("libtiff/4.1.0"),
+        self.requires("libidn/1.36")
+        self.requires("openssl/1.1.1g")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("podofo-{0}".format(self.version), self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+        self._cmake.definitions["PODOFO_BUILD_LIB_ONLY"] = True
+        if self.options.shared:
+            self._cmake.definitions["PODOFO_BUILD_SHARED"] = True
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+
+    def package_info(self):
+        self.cpp_info.filenames["cmake_find_package"] = "podofo"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "podofo"
+        self.cpp_info.names["cmake_find_package"] = "podofo"
+        self.cpp_info.names["cmake_find_package_multi"] = "podofo"
+        self.cpp_info.components["_podofo"].names["cmake_find_package"] = "podofo"
+        self.cpp_info.components["_podofo"].names["cmake_find_package_multi"] = "podofo"
+        self.cpp_info.components["_podofo"].includedirs = ["include"]
+        self.cpp_info.components["_podofo"].libdirs = ["lib"]
+        self.cpp_info.components["_podofo"].libs = ["podofo"]
+        if self.settings.os == "Linux":
+            self.cpp_info.components["_podofo"].system_libs = ["pthread"]
+        self.cpp_info.components["_podofo"].requires = ["freetype::freetype", "libjpeg::libjpeg", 
+            "libunistring::libunistring", "libtiff::libtiff", "libidn::libidn", "openssl::openssl"]
+        if self.settings.os == "Linux":
+            self.cpp_info.components["_podofo"].requires.append("fontconfig::fontconfig")
```

Comment:
```suggestion
        self.cpp_info.filenames["cmake_find_package"] = "podofo"
        self.cpp_info.filenames["cmake_find_package_multi"] = "podofo"
        self.cpp_info.names["pkg_config"] = "libpodofo-0"
        self.cpp_info.names["cmake_find_package"] = "podofo"
        self.cpp_info.names["cmake_find_package_multi"] = "podofo"
        self.cpp_info.libs = ["podofo"]
        if self.settings.os == "Linux":
            self.cpp_info.system_libs = ["pthread"]
```
There is no win in using components when there is only one